tests/images_hierarchy.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+from pathlib import Path
+from typing import Optional
+
+THIS_DIR = Path(__file__).parent.resolve()
+
+
```

Comment:
Can we add a link to https://jupyter-docker-stacks.readthedocs.io/en/latest/using/selecting.html#image-relationships for new contributors?